community/issue_tracking.md

```diff
@@ -1,25 +1,17 @@
----
-layout: page
-hide: true
-title: Tracking Issues
-permalink: /community/issue_tracking/
-# Copyright (c) 2015-2017, Intel Corporation.
-# Licensed under Creative Commons Attribution 4.0 International License
-# https://creativecommons.org/licenses/by/4.0/
----
-
-***************
-Tracking Issues
-***************
+<!--
+ Copyright (c) 2015-2017, Intel Corporation.
+ Copyright (c) 2020, Walmart Inc.
+ Licensed under Creative Commons Attribution 4.0 International License
+ https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Tracking Issues
 
 A great way to contribute is by reporting issues. Before reporting an issue,
 please review the current open issues to see if someone has already reported
 the issue.
 
-.. _jira:
-
-Using JIRA
-==========
+## Using JIRA
```

Comment:
The links in this section need markup, otherwise they are generated as standard text (no link).

Reply:
Thanks, missed it